src/board/v3/nap/nap_common.c

```diff
@@ -21,19 +22,25 @@
 #include "system_monitor.h"
 
 #include <math.h>
+#include <string.h>
 
 #define PROCESS_PERIOD_ms (1000)
 
 void nap_isr(void *context);
+void nap_unlock(const u8 key[]);
 
 static BSEMAPHORE_DECL(nap_exti_sem, TRUE);
 static WORKING_AREA_CCM(wa_nap_exti, 2000);
 static void nap_exti_thread(void *arg);
 
+static u8 nap_dna[8] = {0};
```

Comment:
Are the other offsets / lengths in bytes? Seems more consistent to use bytes for everything.